example/pubspec.yaml

```diff
@@ -11,8 +11,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_lints: ^1.0.4
-  flutter_reactive_ble:
-    path: ../packages/flutter_reactive_ble
+  flutter_reactive_ble: ^5.0.1
```

Comment:
This should also point to the local version

Reply:
The dependency_overrides below should ensure it does. I wasn't sure if both dependencies should be in dependency_overrides like I've made it, or the path should remain here as before. It felt to me more consistent to have both as overrides.